winter_messaging_outbox_transacton/start_worker.py

```diff
@@ -0,0 +1,74 @@
+import argparse
```

Comment:
File name suggestion: `worker.py` or `consumer.py` or the same with prefix `winter_consumer.py`, usually we don't put "start" into the name of the script.